### PR TITLE
fix: presuggestion not working on mobile devices if word start capitalized (AP-254)

### PR DESCRIPTION
**Pull request by nickjjang**
AP-254

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]
fixed ✅

**Review by josephmcg — APPROVED** [review]

File: components/ui/TypeHead/TypeHead.vue [hunk 1]
```diff
@@ -73,15 +84,22 @@ export default Vue.extend({
   },
   methods: {
     update() {
+      if (this.$props.textTransform === 'lowercase') {
```
Comment (WanderingHogan) [inline components/ui/TypeHead/TypeHead.vue:87]: Can you remove the pieces where you force lowercase input and instead favor just casting the input to lowercase to compare with the list? I think you can accomplis this in the update with something similar to ```      this.searchList = this.list.filter((item: any) => { 
        return item.indexOf(this.searchText.toLowerCase()) !== -1
      }```

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
Can you make this so the user can still input any case, but we cast it to lower case when comparing to the list?

**Review by WanderingHogan — APPROVED** [review]
works good!

**Review by stavares843 — APPROVED** [review]

**Pull request merged**